_includes/documentation/documentation-version-band.html

```diff
@@ -33,8 +33,8 @@ <h3>Strimzi {{site.data.releases.operator[0].version}} documentation (latest sta
         <strong class="text-caps">Strimzi Kafka operators</strong>
         <br/>
         <a href="{{site.baseurl}}/docs/operators/in-development/overview.html">Overview</a>  <a href="/docs/operators/in-development/full/overview.html" title="Full documentation (New Window)" target="_blank"><i class="fas fa-external-link-alt"></i></a> |
-        <a href="{{site.baseurl}}/docs/operators/in-development/deploying.html">Deploying and Upgrading</a>  <a href="/docs/operators/in-development/full/deploying.html" title="Full documentation (New Window)" target="_blank"><i class="fas fa-external-link-alt"></i></a> |
-        <a href="{{site.baseurl}}/docs/operators/in-development/configuring.html">Configuring</a>  <a href="/docs/operators/in-development/full/configuring.html" title="Full documentation (New Window)" target="_blank"><i class="fas fa-external-link-alt"></i></a> |
+        <a href="{{site.baseurl}}/docs/operators/in-development/deploying.html">Deploying and Managing</a>  <a href="/docs/operators/in-development/full/deploying.html" title="Full documentation (New Window)" target="_blank"><i class="fas fa-external-link-alt"></i></a> |
+        <a href="{{site.baseurl}}/docs/operators/in-development/configuring.html">Custom Resource API Reference</a>  <a href="/docs/operators/in-development/full/configuring.html" title="Full documentation (New Window)" target="_blank"><i class="fas fa-external-link-alt"></i></a> |
```

Comment:
Would just `API Reference` be more clear? It is still confusing for many users what custom resource is.